### ISSUE TITLE
option to only print red URLs?

### ISSUE THREAD

**Issue by maelle** [urlstechie/urlchecker-python#29]
Could there be an option to only print the URLs for which there was a failure? This way it'd be easier to go through them for a website that has many URLs?

**Comment by vsoch** [comment]
This issue has been transferred to urlchecker-python - it will be followed up with update to urlchecker-action.

**Comment by vsoch** [comment]
@SuperKogito you should be unblocked here now too!

**Comment by SuperKogito** [comment]
yes I will make a PR of this today. Just wanted to ask, how about including a `--prints-style=none`. The idea behind this, is that it can be useful, if we want to implement *multiprocessing* to provide a fast processing option, where the user will be only interested in the output file.
